packages/client/src/pages/olo/OloLocationInput.test.js

```diff
@@ -0,0 +1,138 @@
+import "@testing-library/jest-dom/extend-expect";
+
+import React from "react";
+
+import mocks from "../../__mocks__/address";
+import addressMock from "../../__mocks__/addressMock";
+import Context from "../../__mocks__/context";
+import matchMedia from "../../__mocks__/matchMedia";
+import { findTopicBySlug } from "../../utils";
+import { LOCATION_FOUND } from "../../utils/test-ids";
+import { act, fireEvent, render, screen } from "../../utils/test-utils";
+import OloLocationInput from "./OloLocationInput";
+
+Object.defineProperty(window, "matchMedia", matchMedia);
+
+const matomoTrackEvent = jest.fn();
+window.open = jest.fn();
+window.scrollTo = jest.fn();
+
+jest.mock("react-router-dom", () => ({
+  useHistory: () => ({
+    location: {
+      pathname: "/aanbouw-of-uitbouw-maken/adresgegevens",
+    },
+    push: jest.fn(),
+  }),
+  useLocation: () => jest.fn(),
+  useParams: () => ({ slug: "aanbouw-of-uitbouw-maken" }),
+}));
+
+jest.mock("react-hook-form", () => ({
+  useForm: () => ({
+    handleSubmit: jest.fn(),
+  }),
+}));
+
+window.history.pushState({}, "Locatie pagina", "/dakkapel-plaatsen/locatie");
+
+const mockedFunctions = { ...{ matomoTrackEvent } };
+
+const mockedAddress = {
+  address: {
+    houseNumber: "19c",
+    houseNumberFull: "19c",
+    postalCode: "1055XD",
+  },
+};
```

Comment:
Can you pick an address from the `mocking` package?

Reply:
as discussed. We need to have the exact input to mock the graphql